components/nav.js

```diff
@@ -15,6 +15,7 @@ const Join = () => {
           margin-left: 16px;
           text-decoration: none;
           text-transform: uppercase;
+          color: #fff;
```

Comment:
Need to edit this & it’s in a separate PR…can we remove from here?

Reply:
sure thing